A43-sdk-authorization.md

```diff
@@ -690,9 +684,8 @@ of policy (if the policy is an allowlist or denylist) and whether a matching pol
 was found.
 
 This engine implementation is shared by SDK-based authorization and xDS-based
-authorization. In gRPC Proxyless Service Mesh (or xDS authorization), Istio or
-Traffic Director control plane sends Envoy RBAC policy configs to xDS enabled gRPC
-servers for authorization. ([A41](https://github.com/grpc/proposal/pull/237): xDS RBAC Support)
+authorization. In xDS authorization, the xDS server sends an RBAC policy config to
+xDS-enabled gRPC server for authorization.([A41](https://github.com/grpc/proposal/blob/master/A41-xds-rbac.md): xDS RBAC Support)
```

Comment:
This should be a relative link, just `A41-xds-rbac.md`, not `https://github.com/grpc/proposal/blob/master/A41-xds-rbac.md`.

Reply:
Done.